src/components/Footer/index.tsx

```diff
@@ -1,22 +1,39 @@
 import React from 'react';
 
 import './style.scss';
-import styled from '@datapunt/asc-core';
+import styled from 'styled-components';
 import {
   Footer as AscFooter,
   FooterTop,
   Row,
   Column,
-  FooterToggle,
-  FooterContent,
-  FooterLinkList,
-  FooterLinkListItem,
+  List,
+  ListItem,
   Link,
-  FooterHeading,
-} from '@datapunt/asc-ui';
+  themeColor,
+} from '@amsterdam/asc-ui';
 
 const StyledFooter = styled(AscFooter)`
-  .;
+  svg path {
+    fill: ${themeColor('tint', 'level1')} !important;
```

Comment:
Is this `important` real necessary? It should be possible through a `&& > ` construction

Reply:
unfortunately yes!

the footer should not have white color I think